sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerPage.scala

```diff
@@ -72,6 +72,9 @@ private[ui] class ThriftServerPage(parent: ThriftServerTab) extends WebUIPage(""
     val table = if (numStatement > 0) {
       val headerRow = Seq("User", "JobID", "GroupID", "Start Time", "Finish Time", "Close Time",
         "Execution Time", "Duration", "Statement", "State", "Detail")
+      val tooltips = Seq(None, None, None, None, Some(ToolTips.THRIFT_SERVER_FINISH_TIME),
+        Some(ToolTips.THRIFT_SERVER_CLOSE_TIME), Some(ToolTips.THRIFT_SERVER_EXECUTION),
+        Some(ToolTips.THRIFT_SERVER_DURATION), None, None, None)
```

Comment:
Or, if not inside `UIUtils.listingTable`, maybe add `assert(headerRow.length == tooltips.length)` here and in ThriftServerSessionPage?

Reply:
I've been thinking about it, yes, it's a good idea, in that way we don't break the homogeneous way of treating the parameters at _UIUtils.listingTable_ and we ensure it is going to be less problems if new headers are added